client/src/index.css

```diff
@@ -5,6 +5,7 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  white-space: pre-wrap;
```

Comment:
There is problem with this css property, it breaks some part of the page by adding lot of space. Can you use it only for the <p> tag used to display the document text and translation?